src/pages/quiz/summmary/QuizPage.tsx

```diff
@@ -88,7 +92,16 @@ const QuizPage = ({quizId, workspaceId}: { quizId: string, workspaceId: string }
             navigation.navigate("QuestionsScreen", {
                 quizId: quizId,
                 questions: questions,
-                quiz: quiz
+                quiz: quiz,
+                previouslyCorrect: 0
+            });
+        };
+        const navigateToOnlyIncorrectQuestionScreen = () => {
```

Comment:
it still navigate to the QuestionScreen with all question. I checked it and backend restuns correct numebr of questions but it is not working there